nengo/utils/stdlib.py

```diff
@@ -239,7 +239,7 @@ def get_terminal_size(fallback=(80, 24)):
 
     try:
         return shutil.get_terminal_size(fallback)
-    except Exception:  # pragma: no cover
+    except Exception:  # pragma: no cover pylint: disable = broad-except
```

Comment:
I think we actually did this for backwards compat reasons as `shutil` didn't have `get_terminal_size` until Python 3.3. Will just remove it.